### ISSUE TITLE
1.7.0-pre.4 regression: not handled bare module specifiers correctly

### ISSUE THREAD

**Issue by web-padawan** [Polymer/polymer-cli#993]
### Description
After upgrading to latest polymer-cli 1.7.0-pre.4 released today, we get 404 in tests for component converted using `--import-style=name` and the false positive suite success (with 0 tests).

### Versions & Environment
- Polymer CLI: 1.7.0-pre.4
- node: 8.11.1
- Operating System: Mac OS

#### Steps to Reproduce

1. Clone the repo:
```sh
git clone --depth 1 -b p3-preview-bare git://github.com/vaadin/vaadin-themable-mixin.git && cd vaadin-themable-mixin`
```

2. Install the modulizer:
```sh
git clone git://github.com/Polymer/polymer-modulizer.git && cd polymer-modulizer && npm link
```

3. Install the tools:
```sh
npm i -g yarn magi-cli polymer-cli@next
```

4. Run the conversion:
```sh
magi p3-convert --out . --import-style=name
```

5. Install dependencies:
```sh
yarn installl --flat
```

6. Serve the component:
```sh
polymer serve --npm --module-resolution=node
``` 

7. Go to http://127.0.0.1:8081/components/@vaadin/vaadin-themable-mixin/test/

#### Expected Results

Tests passed in Travis: https://travis-ci.org/vaadin/vaadin-themable-mixin/jobs/363066517 

```
Test run ended with great success
macOS 10.12 iphone 11.2 (10/0/0)      
macOS 10.12 ipad 11.2 (20/0/0)        
Windows 10 chrome 65 (20/0/0)         
macOS 10.12 safari 11.0 (20/0/0) 
```

... and locally:
![screen shot 2018-04-06 at 14 48 55](https://user-images.githubusercontent.com/10589913/38419866-a3d08b50-39aa-11e8-94e1-23ec63bac7ff.png)

#### Actual Results

False positive build in Travis: https://travis-ci.org/vaadin/vaadin-themable-mixin/jobs/363042746

```
Test run ended with great success
macOS 10.12 iphone 11.2 ()            
macOS 10.12 ipad 11.2 ()              
Windows 10 chrome 65 ()               
macOS 10.12 safari 11.0 () 
```

... and locally: 
![screen shot 2018-04-06 at 14 51 25](https://user-images.githubusercontent.com/10589913/38419876-adbff95c-39aa-11e8-99e8-341bd5f31bc8.png)

**Comment by web-padawan** [comment]
UPD: also happens without using bare module specifiers, see also following builds:

Broken (1.7.0-pre.4): https://travis-ci.org/vaadin/vaadin-context-menu/jobs/363065641

Good (1.7.0-pre.3): https://travis-ci.org/vaadin/vaadin-context-menu/jobs/363077464

**Comment by justinfagnani** [comment]
Thanks for the report @web-padawan 

Can you see if this happens when using a "app-style" URL, like http://127.0.0.1:8081/test/ (assuming you're testing @vaadin/vaadin-themable-mixin)

**Comment by web-padawan** [comment]
@justinfagnani in that case I only get 404 for `http://127.0.0.1:8081/wct-browser-legacy/browser.js`

**Comment by justinfagnani** [comment]
That's because of the `../../` style relative URLs in the HTML test files. I updated to `/node_modules/..` URLs and the tests load, though it doesn't look like they're modulized correctly.

**Comment by web-padawan** [comment]
Yes, something is wrong with the modulizer master branch as well. In our team we use the forked version which worked for us so far, including the 1st test suite in the issue. You can install it like this:
```sh
git clone --depth 1 -b vaadin-components git://github.com/web-padawan/polymer-modulizer.git && cd polymer-modulizer && npm link
```

**Comment by justinfagnani** [comment]
What's the difference in that version?

**Comment by web-padawan** [comment]
My fork started at Polymer/polymer-modulizer@e4f3409 and then included some updates:
- updated dependency mapping to 3.0.0-pre.12
- update to analyzer 3.0.0-pre.17

I will also try Polymer/polymer-modulize#392 to see if it helps.

**Comment by aomarks** [comment]
> @aomarks thanks! can you update polymer-cli@next tag to point to that version please?

Done.

**Comment by web-padawan** [comment]
@aomarks BTW seeing lots of console output now:

```sh
$ polymer serve --npm --module-resolution=node
info:    Files in this directory are available under the following URLs
      applications: http://127.0.0.1:8081
      reusable components: http://127.0.0.1:8081/components/@vaadin/vaadin-themable-mixin/

js-transform
js-transform
js-transform
js-transform
js-transform
js-transform
js-transform
js-transform
js-transform
js-transform
js-transform
js-transform
js-transform
[BABEL] Note: The code generator has deoptimised the styling of undefined as it exceeds the max of 500KB.
js-transform
js-transform
```
Not sure where it could be introduced, but looks funny 😃

**Comment by aomarks** [comment]
> @aomarks BTW seeing lots of console output now:

I'm not seeing these when I run polymer serve at 1.7.0-pre.9, but let me know if you still see these, since I was never able to reproduce.
